mmdet/models/backbones/resnet.py

```diff
@@ -279,14 +326,21 @@ def init_weights(self, pretrained=None):
             for m in self.modules():
                 if isinstance(m, nn.Conv2d):
                     kaiming_init(m)
-                elif isinstance(m, nn.BatchNorm2d):
+                elif (isinstance(m, nn.BatchNorm2d)
+                      or isinstance(m, nn.GroupNorm)):
                     constant_init(m, 1)
+
+            # zero init for last norm layer https://arxiv.org/abs/1706.02677
```

Comment:
We can add an argument `zero_init_residual` as in [torchvision](https://github.com/pytorch/vision/blob/master/torchvision/models/resnet.py).

Reply:
I don't see any tradeoff when setting zero_init_residual to False. Since it is recommended, should we use it as defaults?

Reply:
I agree that we can set the default value as `True`.